models/trading.js

```diff
@@ -26,16 +28,17 @@ const getUpdatedPrice = (stockPrice) => {
 const trade = async (tradeData) => {
```

Comment:
@sumitd94 , can we have a tradeService.js and add all the business logic there, and keep only the CRUD logic in models files?

Reply:
Similar to what we do for auth, github and contributions..

Reply:
The entire logic is business here, should I move all the logic into a service file and call this function from the controller?

Reply:
2d78b75

Reply:
Yup 👍